src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -36073,7 +36073,7 @@ static void Main()
             CompileAndVerify(compilation3, expectedOutput: expectedOutput);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(CoreClrOnly), Reason = ConditionalSkipReason.MonoDefaultInterfaceMethods)]
```

Comment:
>typeof(CoreClrOnly) [](start = 25, length = 19)

I think we should keep running these tests on Windows Desktop

Reply:
Doh. Yeah we should. Let me fix.